### PR TITLE
Integration of Danfoss room sensor 014G0160

### PR DESCRIPTION
**Pull request by Quax1507**
XML file and the addition to manufarturer_specific.xml to get Danfoss room sensor supported

### CODE REVIEW

File: config/danfoss/rs.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product xmlns='http://code.google.com/p/open-zwave/'>
+
+        <CommandClass id="67" name="COMMAND_CLASS_THERMOSTAT_SETPOINT" version="1" request_flags="4" override_precision="2" innif="true" base="0">
```
Comment (Fishwaldo) [inline config/danfoss/rs.xml]: Why do you need to add the Thermostat CC in the config file? Does the device not auto-discover correctly?

**Comment by Quax1507** [comment]
You are right. Discovery is incorrect. Do You need any details?

File: config/danfoss/rs.xml (hunk 1 above)
Comment (Quax1507) [inline config/danfoss/rs.xml]: You are right. Discovery is incorrect. Do You need any details?

**Comment by Quax1507** [comment]
You are right. Discovery is incorrect. Do You need any details?

**Comment by alexus1211** [comment]
it uses "COMMAND_CLASS_THERMOSTAT_SETPOINT" version "3" and not version "1" as described in rs.xml.
Did this patch fix the discovery problem? BTW, I have the same problem in Razberry (Zway).

**Comment by Quax1507** [comment]
With that fix, the Thermostat works without any issues!
The discovery returns wrong value "4" for "furnace". Correct vaue is "1" instead!
Will also occur with Zway :-(

**Comment by alexus1211** [comment]
Can you send me the logs for the auto discovery? What I meant with "3" is the version of the Command Class "thermostat_setpoint" that the RS supports, and not the payload data that has been transmitted.

**Comment by Fishwaldo** [comment]
The version reported in the zwcfg_*.xml file is the version that OZW supports, not what the device reports. All CommandClasses are backwards compatible, so if we don't support version 3, it should still work. 

please attach the logs to this ticket after starting OZW without any zwcfg_*.xml file.

**Comment by Fishwaldo** [comment]
@alexus1211 any update?

**Comment by Fishwaldo** [comment]
Closing as no response.

**Pull request closed without merging**